golangci-lint 1.30.0 reports gosec G404 instance (weak RNG) at home/auth.go:279

### Issue Details

* **Version of AdGuard Home server:**
  * 8dc010886867de62340d933df7edc31314234abc

### Expected Behavior
No `golangci-lint` errors (though, it's entirely reasonable here as the repo is using something like 1.27 and I'm reporting a 1.30.0 issue).

### Actual Behavior
```
home/auth.go:279:36: G404: Use of weak random number generator (math/rand instead of crypto/rand) (gosec)
        d := []byte(fmt.Sprintf("%d%s%s", rand.Uint32(), u.Name, u.PasswordHash))
```

### Additional Information
I'd be happy to submit a PR if you'd let me know whether: (A) you'd like to disable the lint check for this occurrence or (B) switch to `crypto/rand`. Looking at the code in question, using `rand.Uint32()` here doesn't seem _obviously_ insecure but I've not thought very deeply about it.

--- a/home/auth.go
+++ b/home/auth.go
@@ -276,7 +276,11 @@ type loginJSON struct {
 }
 
 func getSession(u *User) []byte {
-	d := []byte(fmt.Sprintf("%d%s%s", rand.Uint32(), u.Name, u.PasswordHash))
+	// the developers don't currently believe that using a
+	// non-cryptographic RNG for the session hash salt is
+	// insecure
+	salt := rand.Uint32() //nolint:gosec
+	d := []byte(fmt.Sprintf("%d%s%s", salt, u.Name, u.PasswordHash))
 	hash := sha256.Sum256(d)
 	return hash[:]
 }

